src/scripts/main.js

```diff
@@ -1,3 +1,219 @@
 'use strict';
 
-// write code here
+const tHead = document.querySelector('thead');
+const tbody = document.querySelector('tbody');
+const rowsArray = Array.from(tbody.rows);
+let firstHeaderClick = false;
+
+function sort() {
+  tHead.addEventListener('click', (e) => {
+    const th = e.target;
+
+    const sortTable = function(colNum = 0) {
+      if (firstHeaderClick) {
+        firstHeaderClick = false;
+
+        const compare = function(rowA, rowB) {
+          if (rowA.cells[colNum].innerHTML.charAt(0) === '$') {
+            return rowA.cells[colNum].innerHTML
+              .split('$').join('').split(',').join('.')
+                - rowB.cells[colNum].innerHTML
+                  .split('$').join('').split(',').join('.');
+          }
+
+          return rowA.cells[colNum].innerHTML > rowB.cells[colNum].innerHTML
+            ? 1 : -1;
+        };
+
+        rowsArray.sort(compare);
+      } else {
+        firstHeaderClick = true;
+
+        const compare = function(rowA, rowB) {
+          if (rowA.cells[colNum].innerHTML.charAt(0) === '$') {
+            return rowB.cells[colNum].innerHTML
+              .split('$').join('').split(',').join('.')
+                - rowA.cells[colNum].innerHTML
+                  .split('$').join('').split(',').join('.');
+          }
+
+          return rowA.cells[colNum].innerHTML > rowB.cells[colNum].innerHTML
+            ? -1 : 1;
```

Comment:
```suggestion
          return rowA.cells[colNum].innerHTML > rowB.cells[colNum].innerHTML;
```
true/false преобразуются в числа